.changeset/fair-actors-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+Add duration and sourcemap size to upload metrics event
```

Comment:
This should follow the changeset format: https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets